test/comment_referable_test.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dartdoc.comment_reference_test;
+
+import 'package:analyzer/dart/element/element.dart';
+import 'package:dartdoc/src/model/comment_referable.dart';
+import 'package:dartdoc/src/model/nameable.dart';
+import 'package:test/test.dart';
+
+const SEPARATOR = '.';
```

Comment:
FWIW Effective Dart says to prefer lowerCamel, unless SCREAMING_CAPS is consistent with other variables. https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names So this could go either way.